### PR TITLE
Add Sentry

### PR DESCRIPTION
**Pull request by RoyEJohnson**

### CODE REVIEW

File: src/app/sentry.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import * as Sentry from '@sentry/react';
+import {Integrations} from '@sentry/tracing';
+
+Sentry.init({
+    dsn: 'https://8468b55bf5b2442d9d31840113399b10@o484761.ingest.sentry.io/5691260',
```
Comment (nathanstitt) [inline src/app/sentry.js:5]: should this also be read from env?

**Review by nathanstitt — APPROVED** [review]

File: src/app/sentry.js (hunk 1 above)
Comment (RoyEJohnson) [reply src/app/sentry.js:5]: I don't *think* so, but this was my first experience with Sentry.

Comment (mwharrison) [reply src/app/sentry.js:5]: @RoyEJohnson it would be good to put this in the env - although it's not super secret, it can still be used nefariously.

Comment (mwharrison) [reply src/app/sentry.js:5]: note: you can generate a new key easily (and should) here: https://sentry.io/settings/openstax/projects/osweb/keys/

Comment (RoyEJohnson) [reply src/app/sentry.js:5]: @mwharrison Is pu something I need Scott to set up? I create the key, have him put it into the environment?

Comment (mwharrison) [reply src/app/sentry.js:5]: @RoyEJohnson I'm not sure how the FE reads secrets from the env - do you have anything else that does this? @nathanstitt how is tutor doing this?

**Pull request merged**